packages/stencil-store/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stencil/store",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
There is a breaking change, I would say this is a 0.3.0

Reply:
i think this ia a previous release, we never updated the package.json my fault